### ISSUE TITLE
custom properties added to copies in 3.1.0 not displayed in 3.2.0

### ISSUE THREAD

**Issue by godmar** [ct-js/ct-js-old#408]
**Describe the bug**
Custom properties added to copies in 3.1.0 aren't displayed in 3.2.0

**To Reproduce**
1. Create a project in 3.1.0
2.  Add a room
3.  Add a template
4. Add a copy to the room
5. Add a custom property to the copy
6. Save
7. Open the project in 3.2.0
8. Custom property is gone

**Expected behavior**
The custom prop should stay

**System information report**
Ct.js v3.2.0 😽 (packaged)

NW.JS v0.67.1
Chromium v104.0.5112.80
Node.js v18.7.0
Pixi.js v5.3.11

WebGL available
WebGL vendor Google Inc. (Google)
WebGL renderer ANGLE (Google, Vulkan 1.2.0 (SwiftShader Device (Subzero) (0x0000C0DE)), SwiftShader driver)
OS linux x64 // Linux 6.1.4-060104-generic

![Screenshot from 2023-01-21 22-12-55](https://user-images.githubusercontent.com/486389/213898923-0e5b49b6-e3b8-4dcd-aa44-dab12ed715fe.png)

**Example project (strongly recommended)**
Upload a minimal zipped project that showcases the bug.

[testcustombug.zip](https://github.com/ct-js/ct-js/files/10473647/testcustombug.zip)

**Comment by godmar** [comment]
Possible fix:
```diff
diff --git a/src/riotTags/rooms/room-entities-properties.tag b/src/riotTags/rooms/room-entities-properties.tag
index 19a60679..8e8146b2 100644
--- a/src/riotTags/rooms/room-entities-properties.tag
+++ b/src/riotTags/rooms/room-entities-properties.tag
@@ -72,7 +72,7 @@ room-entities-properties
         // Custom properties for copies
         div(if="{hasCopies}")
             h3 {voc.customProperties}
-            table.wide.aPaddedTable.cellsmiddle(if="{changes.customProps?.length}")
+            table.wide.aPaddedTable.cellsmiddle(if="{Object.entries(changes.customProps).length}")
                 tr
                     th {voc.copyCustomProperties.property}
                     th {voc.copyCustomProperties.value}
```

I think 6a9c21ff3 may be incorrect.

**Comment by godmar** [comment]
Also, the current interface does
```
tr(each="{val, prop in changes.customProps}")
```
which randomly resorts the list of properties as soon as you add a new one.

**Comment by CosmoMyzrailGorynych** [comment]
Fixed in 84b153995e92ca11fbfe170d937c84739f488aa6
